aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -1375,6 +1387,19 @@ async def apply_constraint_received_cred(
                         return False
         return True
 
+    def get_dict_keys_from_path(self, derived_cred_dict: dict, path: str) -> list:
+        """Return list of keys as additional_attrs to build nested_field_paths."""
+        if path:
+            return self.get_dict_keys_from_path(derived_cred_dict[path[0]], path[1:])
```

Comment:
`path` does not seem to be guaranteed to have multiple entries, for example given a path with no dots in it.